.syncpackrc

```diff
@@ -1,22 +1,46 @@
 {
 	"dev": true,
```

Comment:
FYI these properties changed in [9.0.0](https://github.com/JamieMason/syncpack/releases/tag/9.0.0) @samueljseay

```diff
- "dev": true,
"filter": "^(?:postcss|postcss-loader|pnpm|node|config|react|react-dom|eslint|typescript|@types/react|@types/wordpress__components|@types/node|@(?:typescript-eslint|wordpress)/.*)$",
- "indent": "\t",
- "overrides": true,
- "peer": true,
- "pnpmOverrides": true,
- "prod": true,
- "semverRange": "^",
- "workspace": true,
+ "dependencyTypes": ["dev", "overrides", "peer", "pnpmOverrides", "prod", "workspace"]
```

I'm not sure what behaviour you'll be getting at the moment, it might be checking everything. Although the only thing excluded is `resolutions` which you're probably not using given that's from yarn, so you could omit `dependencyTypes` altogether which would include everything by default.

A 2nd thing is that you could remove that `filter` regex and replace it with 2 new entries at the end of your `versionGroups` array if you would prefer that – it may or may not be easier to follow for you. If I understand right, you don't want to sync the versions for any dependencies in your monorepo other than the allow list in that regexp?

```jsonc
[
  // your current version groups above...
  {
    "label": "Only manage versions for these dependencies",
    "dependencies": [
      "@types/node",
      "@types/react",
      "@types/wordpress__components",
      "@typescript-eslint/**",
      "@wordpress/**",
      "config",
      "eslint",
      "node",
      "pnpm",
      "postcss",
      "postcss-loader",
      "react",
      "react-dom",
      "typescript"
    ],
    "packages": ["**"]
  },
  {
    "label": "Ignore versions for everything else",
    "dependencies": ["**"],
    "packages": ["**"],
    "isIgnored": true
  }
]
```

- https://jamiemason.github.io/syncpack/config/dependency-types
- https://jamiemason.github.io/syncpack/config/filter
- https://jamiemason.github.io/syncpack/config/version-groups

Reply:
Thanks @JamieMason I really appreciate you looking in on these PRs!

🙇🏻